notebooks/dash/dash.ipynb

```diff
@@ -754,18 +754,18 @@
     "\n",
     "**Authors:** Catherine Martlin; WFC3 Instrument Team\n",
     "\n",
-    "**Updated on:** 2021-10-07\n",
+    "**Updated on:** 2023-01-25\n",
     "\n",
     "<a id=\"cite\"></a>\n",
     "## Citations\n",
     "\n",
-    "If you use `numpy`, `matplotlib`, `astropy`, `astroquery`, or `drizzlepac` for published research, please cite the\n",
-    "authors. Follow these links for more information about citing the libraries below:\n",
+    "If you use `numpy`, `matplotlib`, `astropy`, `astroquery`, `photutils`, or `drizzlepac` for published research, please cite the authors. Follow these links for more information about citing the libraries below:\n",
     "\n",
     "* [Citing `numpy`](https://www.scipy.org/citing.html#numpy)\n",
```

Comment:
Numpy has a new citation link: https://numpy.org/citing-numpy/

Reply:
Ohhh, I'm so glad you said that about the format of the section titles - I think it's something in my jupyter notebook settings and it happened automatically and I think it's not pretty at all but thought it was purposeful as I hadn't changed anything manually and was going to leave it be. 

I'll go check and see if I can get it to reset to not have all those numbers and to match the other notebooks formatting.